images/kubekins-e2e/Dockerfile

```diff
@@ -85,16 +85,6 @@ ARG OLD_BAZEL_VERSION
 COPY --from=old \
     /usr/local/lib/bazel/bin/bazel-real /usr/local/lib/bazel/bin/bazel-${OLD_BAZEL_VERSION}
 
-# if UPGRADE_DOCKER_ARG, then install the latest docker over whatever we have
-# in the base image.
-# TODO: after code freeze, roll out newer docker in the
-# base image and possibly remove this ...
-ARG UPGRADE_DOCKER_ARG=false
```

Comment:
toDONE.

we should upgrade bootstrap instead, installing it twice wastes time and space and bootstrap updates need to be done carefully

long term we should eliminate the bootstrap stuff and consolidate down to a single dockerfile for kubekins